fastqfil.py

```diff
@@ -0,0 +1,57 @@
+from additional_scripts.check_of_procedure import check_of_procedure
+from additional_scripts.type_of_chain import type_of_chain
+from additional_scripts.transcribe import transcribe
+from additional_scripts.reverse import reverse
+from additional_scripts.complement import complement
+from additional_scripts.reverse_complement import reverse_complement
+
+'''Accepts dictionaries with the sequence name,
+sequence and its quality of reading,
+filters according to the specified parameters
+of the percentage of GC nucleotides,
+the length of the sequences and the average quality 
+of reading. Returns a filtered dictionary'''
+def filter_fastq(seqs, gc_bounds = (0, 100), length_bounds = (0, 2**32), quality_threshold = 0):
+    filtered_fastq = dict()
+    if type(gc_bounds) == int:
+        gc_bounds = (0, gc_bounds)
+    if type(length_bounds) == int:
+        length_bounds = (0, length_bounds)
+    for name, sequal in seqs.items():
+        sequence, quality = sequal[0], sequal[1]
+        if length_bounds[0] <= len(sequence) <= length_bounds[1]:
+            if gc_bounds[0] <= (sequence.count("C") + sequence.count("G")) / len(sequence) * 100 <= gc_bounds[1]:
+                median_quality = 0
+                for symbol in quality:
+                    median_quality += ord(symbol) - 33
+                median_quality = median_quality/len(sequence)

```

Comment:
```suggestion
                median_quality = median_quality / len(sequence)

```

операнды принято разделять пробелами

-0.5